Readme file improvements (#37)
WordPress/props-bot-action: README.md

## Overview

This GitHub Action Helps with collecting contributors associated with a pull request by commenting with a formatted list of contributors.

> For a full breakdown of the project's Props policies, please consult the [Making WordPress Core Handbook](https://make.wordpress.org/core/handbook/best-practices/contributor-attribution-props/).

Reviewer: ```suggestion
For a full breakdown of the WordPress project's Props best practices, please consult the [Making WordPress Core Handbook](https://make.wordpress.org/core/handbook/best-practices/contributor-attribution-props/).
```

## Configuration

### Required configurations
| Key | Default         | Description                                                  |
| --- |-----------------|--------------------------------------------------------------|
| `token` | `$GITHUB_TOKEN` | GitHub token with permission to comment on the pull request. |

## Example Workflow File

To get started, copy and commit the [`example-props-bot.yml` file](https://github.com/WordPress/props-bot-action/blob/trunk/example-props-bot.yml) into the `.github/workflows` directory of your project's repository.

> The example file is generously documented, making it easy to implement and adjusted to suit the needs of your project.

Reviewer: ```suggestion
The example file is generously documented so it can be implemented and adjusted to suit the needs of your project.
```
